src/components/InputField/InputField.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div
+    class="input-field group flex flex-row items-center gap-4 hover:bg-dark-charcoal-06 focus-within:bg-dark-charcoal-06 group-hover:bg-dark-charcoal-06 h-12 p-0.5px focus-within:p-0 border focus-within:border-1.5 border-dark-charcoal-20 rounded-sm overflow-hidden focus-within:border-pink"
+    :class="[
+      {
+        // Padding is set to 1.5px to accommodate the border that will appear later.
+        'border-s-0 ps-1.5px rounded-s-none': connectionSides.includes('start'),
+        'border-e-0 pe-1.5px rounded-e-none': connectionSides.includes('end'),
+      },
+    ]"
+  >
+    <label class="flex-grow" :for="fieldId">
+      <span v-if="labelText" class="sr-only">{{ labelText }}</span>
```

Comment:
Thanks for addressing this @sarayourfriend I forgot to mention the label in my review. I think your proposed solution makes sense.

Reply:
Making `labelText` required does make sense to prevent inaccessible usage. Thanks!

My main argument against the `hideLabelFromVision` prop is that in the Design Library there are no inputs with labels at all so
- that prop would always be `true`
- there are no examples of how the labels would be styled

So I went ahead and hardcoded the label as `.sr-only`.

Reply:
Made `labelText` required in 61687b6. Also separated the `<label>` from the `<input>` by updating ESLint rules in 447a973. As there are no custom labels in the designs, we are good for now. We can implement an option to hide labels if and when the need arises.